src/app/utils/highlights.ts

```diff
@@ -122,7 +201,7 @@ type OnProgress = ({ loaded, total }: OnProgressProps) => void;
 
 type UploadToCloudinaryProps = {
   file: Blob;
-  onProgress: OnProgress;
+  onProgress?: OnProgress;
```

Comment:
What's the reason for this change?

Reply:
Cause I actually give no onProgress function to the `uploadHighlight` and the `uploadToCloudinary` function.